project/Dependencies.scala

```diff
@@ -32,21 +32,20 @@ object Dependencies {
 
     val AkkaProjectionVersionInDocs = "1.6"
 
-    val AkkaPersistenceCassandra = "1.3.0"
-    val AkkaPersistenceJdbc = "5.5.0"
+    val AkkaPersistenceCassandra = "1.3.2"
+    val AkkaPersistenceJdbc = "5.5.2"
 
-    val AkkaPersistenceR2dbc = "1.3.6"
+    val AkkaPersistenceR2dbc = "1.3.7"
     val AkkaPersistenceR2dbcVersionInDocs = VersionNumber(AkkaPersistenceR2dbc).numbers match {
       case Seq(major, minor, _*) => s"$major.$minor"
     }
 
-    val AkkaPersistenceDynamodb = "2.0.5"
+    val AkkaPersistenceDynamodb = "2.0.6"
     val AkkaPersistenceDynamodbVersionInDocs = VersionNumber(AkkaPersistenceDynamodb).numbers match {
       case Seq(major, minor, _*) => s"$major.$minor"
     }
 
-    val alpakkaKafka = sys.props.getOrElse("build.alpakka.kafka.version", "7.0.0")
-    val slick = "3.5.2"
+    val slick = "3.5.1"
```

Comment:
Was rolled back in akka-persistence-jdbc since slick 3.5.2 was an accidental release